canvas_modules/common-canvas/src/common-properties/controls/expression/expression-builder/expression-select-field-function.jsx

```diff
@@ -265,7 +270,37 @@ export default class ExpressionSelectFieldOrFunction extends React.Component {
 		return contentObject;
 	}
 
+	createAddButtonContent(field, index) {
```

Comment:
Do we need 2 functions here that do almost the exact same thing?  We could either pass in the function to use or pass in details so we know what function to use.